### DISCUSSION TITLE
Installation Error

### DISCUSSION THREAD

**Discussion by ghost** [facebookresearch/ParlAI#4214] [General]
Hi, I am trying to install the latest Git version on a Ubuntu 20.4 and am getting the following message after running the command:

`sudo python3 setup.py develop`

`Processing dependencies for parlai==1.5.1 error: importlib-metadata 4.2.0 is installed but importlib-metadata>=4.4; python_version < "3.10" is required by {'markdown'} `

Any help is appreciated.

**Comment by valmirosjunior** [comment]
Hi @ashwinkots I made this change in the requirements.txt

importlib-metadata<4.3 -> importlib-metadata>=4.4

and added this line In the top of it

markdown==3.3.4

And works, for me.

**Reply by stephenroller** [reply]
Hi Valmir, would you be willing to submit a patch?

**Reply by valmirosjunior** [reply]
Sorry man, I didn't see your reply, do you think it's necessary yet?

**Reply by valmirosjunior** [reply]
I think this PR: https://github.com/facebookresearch/ParlAI/pull/4224 has solved the problem.

**Comment by ghost** [comment]
Thanks all. This worked perfectly.
________________________________
